plugins/Sf2Player/Sf2Player.cpp

```diff
@@ -1120,7 +1147,7 @@ void Sf2InstrumentView::modelChanged()
 
 
 	connect( k, SIGNAL( fileChanged() ), this, SLOT( updateFilename() ) );
-
+	connect( k, SIGNAL( patchChanged() ), this, SLOT( updatePatchName() ) );
```

Comment:
I would rather we switch this to new Qt style connect.

Reply:
Done, updated all the remaining occurrences within this pull request.